package.json

```diff
@@ -78,6 +79,7 @@
     "@storybook/testing-library": "^0.0.9",
     "@types/jest": "^27.4.1",
     "@types/json-server": "^0.14.4",
+    "@types/lodash.clonedeep": "^4.5.7",
```

Comment:
Why is this dep in twice? maybe remove this one.

Reply:
hum... it's types